lang/R/inst/extdata/airr-schema.yaml

```diff
@@ -2046,10 +2104,14 @@ Diagnosis:
                         id: DOID:4
                         label: disease
         disease_length:
-            type: string
+            $ref: '#/PhysicalQuantity'
```

Comment:
I understand that duration of time is also a physical quantity, but would it not be worthwhile to make this explicit and use `TimeInterval` instead (even if then `min==max`)?

Reply:
Created TimeQuantity and made the change